### PR TITLE
fixed styling/spacing in the header and text area

### PR DESCRIPTION
**Pull request by Ephraim-Winston**
I worked on some of the styling.

### CODE REVIEW

**Review by FrancescaImmediato — APPROVED** [review]

File: dev-duckies-sm-proj/src/app-wide-components/create-post.jsx
Comment (bflows) [inline dev-duckies-sm-proj/src/app-wide-components/create-post.jsx:1]: Cool responsive create-post component!

**Review by bflows — APPROVED** [review]
Looks great! I approve this PR.

File: dev-duckies-sm-proj/src/app-wide-components/create-post.jsx [hunk 1]
```diff
@@ -1,10 +1,45 @@
...
+          {/*Bellow is navbar that will also contain user pic and name*/}
+            <div className="card">
+             
+                
+                <header className="card-header is-flex-direction-row is-align-items-center p-4">
+                 
+                  <figure className= "image is-48x48 is-square mr-5 ml-3">
+                      <img className= "is-rounded" src = "https://mediaproxy.salon.com/width/1200/https://media2.salon.com/2011/05/ryan_gosling.jpg"></img>
+                    </figure>
+                      <p>Ryan-gooseling-445</p>
```
Comment (Jose-Lema) [inline dev-duckies-sm-proj/src/app-wide-components/create-post.jsx:16]: Very clever. He is Kenough.

File: dev-duckies-sm-proj/src/app-wide-components/create-post.jsx [hunk 2]
```diff
@@ -1,10 +1,45 @@
...
+               {/*Bellow is text form in order to write and submit post*/}
+              <form className="text-form">
+
+
+             
+                <textarea className= "textarea is-primary  is medium" type="text" placeholder = "Share your thoughts..."></textarea>
+               
+              </form>
+               
+                <button className="button is-primary">Share</button>
```
Comment (Jose-Lema) [inline dev-duckies-sm-proj/src/app-wide-components/create-post.jsx:30]: Consider moving the button up in the form so the spacing isn't too far from the text area

**Review by Jose-Lema — APPROVED** [review]
Great job on the component. Sorry for the late review!

**Pull request merged**